src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -261,6 +270,19 @@ class PeerActor(
 
       case GetStatus =>
         sender() ! StatusResponse(Handshaked)
+
+      case StartFastSync(targetHash) =>
+        //todo get storage form somewhere?
+        val storage = FastSyncActor.Storage(
+          new BlockHeadersStorage(EphemDataSource()),
+          new BlockBodiesStorage(EphemDataSource()),
+          new ReceiptStorage(EphemDataSource()),
+          new MptNodeStorage(EphemDataSource()),
+          new EvmCodeStorage(EphemDataSource())
+        )
+        val fastSyncActor = context.actorOf(FastSyncActor.props(self, storage), UUID.randomUUID().toString)
+        context watch fastSyncActor
```

Comment:
why does PeerActor need to watch fastSyncActor? `handleTerminated` should be updated to consider that (currently it assumes it's rlpxConnection that is being terminated).